dummy_oauth.sh

```diff
@@ -0,0 +1,12 @@
+echo " -------------- DUMMY OAUTH -------------- "
```

Comment:
Let's document this script and its usage in an appropriate place (perhaps build/dev/standalone_instance.md, though in the script might work instead) so that others can use it without reverse-engineering what it does.